ensemble-dss/src/main/java/hec/dss/ensemble/DssDatabase.java

```diff
@@ -439,7 +441,33 @@ public void write(hec.metrics.MetricCollectionTimeSeries[] metricsArray) throws
         CatalogIsUpToDate = false;
     }
 
+    public static TimeSeriesContainer[] toTimeSeriesContainer(MetricCollectionTimeSeries mcts) {
+        List<TimeSeriesContainer> tscList = new ArrayList<>();
+        for (MetricCollection mc : mcts) {
+            String stats = mc.getMetricStatistics();
+            String[] statsAsSeparateTimeseries = stats.split("\\|");
 
+            for (int i = 0; i < statsAsSeparateTimeseries.length; i++) {
+                TimeSeriesContainer tsc = getTimeSeriesContainer(mcts.getTimeSeriesIdentifier(), mc, statsAsSeparateTimeseries[i], mc.getValues()[i]);
+                tscList.add(tsc);
+            }
+        }
+        return tscList.toArray(new TimeSeriesContainer[0]);
+    }
+
+    private static TimeSeriesContainer getTimeSeriesContainer(RecordIdentifier identifier, MetricCollection mc, String stat, float[] values) {
```

Comment:
Is this getting a timeseries container from a DSSDatabase, or creating a timeseries container? Might rethink the name. Maybe this belongs as a constructor on TimeSeriesContainer?

Reply:
This is converting MetricCollectionTimeSeries to TimeSeriesContainer[] 

How about the name convertToTimeSeriesContainer  ?

Reply:
I approve. That seems like a good, descriptive name.